modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/tests/RaylifeAPApplicationsStatus.testcase

```diff
@@ -48,6 +46,105 @@ definition {
 		}
 	}
 
+	@description = "Story ID: LPS-159688 - Verify if the details of the Summary on the Bound application view appears."
+	@priority = 4
+	test AssertCanOpenBoundAppDetails {
+		property test.name.skip.portal.instance = "RaylifeAPApplicationsStatus#AssertCanOpenBoundAppDetails";
+
+		// Skip portal instance due to LRQA-79129. Unskip after it is resolved
+
+		task ("Given the Applications page is open") {
+			RaylifeAPNavigator.gotoSidebarItem(itemName = "Applications");
+		}
+
+		if (IsElementPresent(key_product = "Auto", key_status = "Bound", locator1 = "RaylifeAP#APPLICATIONS_NUMBER_WITH_STATUS")) {
+			task ("When the search term is present") {
+				RaylifeAP.search(searchTerm = "AP");
+
+				RaylifeAP.clickInApplicationNumber(
+					product = "Auto",
+					status = "Bound");
+			}
+
+			task ("Then assert the Detail view page and go back to 'Test Test' User") {
+				AssertElementPresent(
+					key_tabTitle = "App Details",
+					locator1 = "RaylifeAP#TAB_TITLE");
+
+				Navigator.openSpecificURL(url = "http://localhost:8080/home");
+
+				User.logoutPG();
+
+				User.firstLoginPG();
+			}
+		}
+		else {
+			task ("Given the new Application flow is open") {
```

Comment:
`Given, And, When, And, Then` A test, an intention. You can use `And` several times in your tasks, but `Given, When and Then` only once

Reply:
Done